migration_steps/shared/helpers.py

```diff
@@ -261,17 +261,13 @@ def get_config(env="local"):
     return config
 
 
-def get_s3_session(session, environment, host, ci="false", account=None):
-    # todo refactor and combine multiple instances of get_s3_session
-    s3_session = session
+def get_s3_session(environment, s3_url, ci="false", account=None):
+    s3_session = boto3.session.Session()
+
     if environment == "local":
-        if host == "localhost":
```

Comment:
this config was in there to run the scripts from outside of containers. We don't bother doing that much these days but it may still be useful as when you're running from not within the docker network then localstack is actually a call to localhost on a specific port. It's the fact that you are within the docker network that allows you to access named containers :-)

Reply:
I think this is probably of limited use, as (as you say) we rarely if ever run these scripts outside docker. So I'd remove it.